bodhi/consumers/masher.py

```diff
@@ -249,6 +250,9 @@ def __init__(self, release, request, updates, agent,
         }
         self.success = False
 
+        reload(sys)
```

Comment:
This seems a bit unconventional. It seems that you have this call here because ```setdefaultencoding()``` gets removed from ```sys``` after the site module runs. However, the API docs for that function warn that it isn't meant to be used by mere mortals:

https://docs.python.org/2.7/library/sys.html#sys.setdefaultencoding

Being new to Bodhi, I'm not sure how the masher is run. Is it launched by fedmsgs? If so, what process listens to that fedmsg? It might be better to have that process set a locale in the environment for this process.